File: prextras-server/src/main/java/com/nicologies/prextras/RunTypePropertiesProcessor.java
package com.nicologies.prextras;

import jetbrains.buildServer.serverSide.InvalidProperty;
import jetbrains.buildServer.serverSide.PropertiesProcessor;

import java.util.Collection;
import java.util.Map;
import java.util.Vector;

public class RunTypePropertiesProcessor implements PropertiesProcessor {
    public Collection<InvalidProperty> process(Map<String, String> properties) {
        return new Vector<InvalidProperty>();
    }
}

File: prextras-server/src/main/kotlin/com/nicologies/prextras/RunTypePropertiesProcessor.kt
package com.nicologies.prextras

import jetbrains.buildServer.serverSide.InvalidProperty
import jetbrains.buildServer.serverSide.PropertiesProcessor
import java.util.Vector

class RunTypePropertiesProcessor : PropertiesProcessor {
    override fun process(properties: Map<String, String>): Collection<InvalidProperty> {
        return Vector()
    }
}